website/utils/renderer.ts

```diff
@@ -0,0 +1,15 @@
+import createEmotionServer from '@emotion/server/create-instance';
+import { cache } from '@emotion/css';
+
+export const renderStatic = async (callback: Function) => {
```

Comment:
`Function` will make `html` be typed as `any`
```suggestion
export const renderStatic = async (callback: () => string) => {
```

Reply:
👍